sdk/core/azure-core-amqp/src/models/messaging_values.cpp

```diff
@@ -9,41 +9,76 @@
 namespace Azure { namespace Core { namespace Amqp { namespace Models { namespace _internal {
   Azure::Core::Amqp::Models::AmqpValue Messaging::DeliveryAccepted()
   {
-    return messaging_delivery_accepted();
+    auto rv = messaging_delivery_accepted();
+    if (!rv)
+    {
+      throw std::runtime_error("Could not allocate delivery accepted described value.");
+    }
+    return rv;
   }
   Azure::Core::Amqp::Models::AmqpValue Messaging::DeliveryReleased()
   {
-    return messaging_delivery_released();
+    auto rv = messaging_delivery_released();
+    if (!rv)
+    {
+      throw std::runtime_error("Could not allocate delivery accepted described value.");
+    }
+    return rv;
   }
   Azure::Core::Amqp::Models::AmqpValue Messaging::DeliveryReceived(
       uint32_t sectionNumber,
       uint64_t sectionOffset)
   {
-    return messaging_delivery_received(sectionNumber, sectionOffset);
+    auto rv = messaging_delivery_received(sectionNumber, sectionOffset);
+    if (!rv)
+    {
+      throw std::runtime_error("Could not allocate delivery accepted described value.");
+    }
+    return rv;
   }
   Azure::Core::Amqp::Models::AmqpValue Messaging::DeliveryRejected(
       std::string const& errorCondition,
       std::string const& errorDescription)
   {
-    return messaging_delivery_rejected(
+    auto rv = messaging_delivery_rejected(
         errorCondition.empty() ? nullptr : errorCondition.c_str(),
         errorDescription.empty() ? nullptr : errorDescription.c_str());
+    if (!rv)
+    {
+      throw std::runtime_error("Could not allocate delivery accepted described value.");
+    }
+    return rv;
   }
   Azure::Core::Amqp::Models::AmqpValue Messaging::DeliveryModified(
       bool deliveryFailed,
       bool undeliverableHere,
       Azure::Core::Amqp::Models::AmqpValue annotations)
   {
-    return messaging_delivery_modified(deliveryFailed, undeliverableHere, annotations);
+    auto rv = messaging_delivery_modified(deliveryFailed, undeliverableHere, annotations);
+    if (!rv)
+    {
+      throw std::runtime_error("Could not allocate delivery accepted described value.");
```

Comment:
Copy/paste error in exception message. Each method should have a specific error message.

Applies to CreateSource, CreateTarget, etc.